pkg/controller/networkpolicy/networkpolicy_controller_test.go

```diff
@@ -3271,6 +3274,58 @@ func TestGetAddressGroupMemberSet(t *testing.T) {
 	}
 }
 
+func TestAddNodeSelector(t *testing.T) {
+	stopCh := make(chan struct{})
+	defer close(stopCh)
+	_, c := newController()
+	c.informerFactory.Start(stopCh)
+	c.crdInformerFactory.Start(stopCh)
+	go c.groupingController.Run(stopCh)
+	go c.groupingInterface.Run(stopCh)
+	c.informerFactory.WaitForCacheSync(stopCh)
+	c.crdInformerFactory.WaitForCacheSync(stopCh)
+	cache.WaitForCacheSync(stopCh, c.groupingInterfaceSynced)
+
+	nodeSelectorA := metav1.LabelSelector{MatchLabels: map[string]string{"env": "pro"}}
+	nodeSelectorB := metav1.LabelSelector{MatchLabels: map[string]string{"env": "test"}}
```

Comment:
nodeSelectorB doesn't seem useful.
Perhaps it could just create two Nodes with Labels and init the nodeSelector with Node1's labels.

Reply:
Done.